Makefile

```diff
@@ -1,5 +1,3 @@
-.PHONY: ui vendor build container-manager container-manager-push push container-frontend container-frontend-push container-backend container-backend-push
```

Comment:
ui build push targets didn't exist anymore, moved the existing ones with the respective target to make it easier to keep these in sync (more easy to detect during a review).